### PR TITLE
Add pagination on overview page

### PR DESCRIPTION
**Pull request by ekhomyakova**
Depends on kqueen/pull/312

### CODE REVIEW

**Comment by ekhomyakova** [comment]
image

**Comment by katyafervent** [comment]
I suggest to reduce the rectangle, that shows active page and make it no bigger than capital letter

File: kqueen_ui/blueprints/manager/views.py [hunk 1]
```diff
@@ -59,35 +73,62 @@ def handle(self):
...
+        return render_template('manager/overview.html',
+                               organizations=organizations,
+                               c_page=get_page(request.args, 'c_page'),
+                               p_page=get_page(request.args, 'p_page'))
 
 
 class DataClusters(KQueenView):
     decorators = [superadmin_required]
     methods = ['GET']
+    objects_per_page = 20
```
Comment (katyafervent) [inline kqueen_ui/blueprints/manager/views.py]: shouldn't we allow user to set up this value in the config file?

File: kqueen_ui/blueprints/manager/views.py [hunk 2]
```diff
@@ -13,6 +13,8 @@
 from kqueen_ui.utils.wrappers import superadmin_required
 from slugify import slugify
 
+from ...blueprints.ui.utils import form_page_ranges
```
Comment (katyafervent) [inline kqueen_ui/blueprints/manager/views.py]: Can we avoid relative imports? it makes impossible test execution not from project directory location

**Pull request merged**